modules/transport.js

```diff
@@ -518,7 +523,31 @@ Transport.prototype.onNewBlock = function (block, broadcast) {
  */
 Transport.prototype.onMessage = function (msg, broadcast) {
 	if (broadcast && !__private.broadcaster.maxRelays(msg)) {
-		__private.broadcaster.broadcast({limit: constants.maxPeers, dappid: msg.dappid}, {api: '/dapp/message', data: msg, method: 'POST', immediate: true});
+		__private.broadcaster.broadcast({limit: constants.maxPeers, dappid: msg.dappid}, {
+			api: '/dapp/message',
+			data: msg,
+			method: 'POST',
+			immediate: true
+		});
+	}
+};
+
+/**
+ * Broadcasts the passed block using __private.broadcaster.broadcast
+ * @implements {Broadcaster.broadcast}
+ * @param {Object} params
+ * @param {Object} block
+ */
+Transport.prototype.broadcastBlock = function (params, block) {
```

Comment:
New function, kind of nice, but what use that function only in 1 place?

Reply:
As broadcaster is initialized only in `transport` module (and should be initialized only once per application as it runs synchronously actions), a public function is the simplest way to access it.

Reply:
I mean there are duplicated code parts inside transport module - so we can use that public function inside transport, or move body to private function + use private function in transport + expose that function here.

Reply:
The function is now reused.